### PR TITLE
Fix autoconf

### PR DESCRIPTION
**Pull request by tru7**
@2franix with this commits also autoreconf should work fine. Please merge.

### CODE REVIEW

**Comment by 2franix** [comment]
Sorry for the delay, I did not get the notification for this one... I will take care of it ASAP.

**Comment by tru7** [comment]
did you check your notification settings in your profile? Maybe you need another address for receiving notifications. Or you may have to check your spam filter.

**Comment by 2franix** [comment]
Yes, settings look good. And I don't have a spam filter at all. 
I am gonna be cautious for some time and hope that was just bad luck

File: Makefile.in [hunk 1]
```diff
@@ -78,7 +78,7 @@ POST_UNINSTALL = :
 subdir = .
 DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/Makefile.am \
 	$(top_srcdir)/configure $(am__configure_deps) \
-	$(srcdir)/config.h.in AUTHORS ChangeLog NEWS README compile \
```
Comment (2franix) [inline Makefile.in]: It is strange that README.md does not appear. Or you had no README.md at the time you generated this file?

**Comment by tru7** [comment]
The point is that README.md cannot be controlled by autoconf because it is unknown, so it will not be mentioned here for dependency checking. README is not mentioned anymore because it does not exist anymore and I switched off gnu checking so it does not complain about missing README.

**Comment by 2franix** [comment]
Got it, thanks. A simple workaround would be to create a symlink README 
-> README.md. I'll take care of that.

**Pull request merged**